app/static/css/tutors.css

```diff
@@ -0,0 +1,61 @@
+.tutor-list-header {
+    width: 100%;
+    color: var(--secondary-color);
+    font-size: 1.5rem;
+    padding: 30px 25% 30px 25%;
+}
+
+.tutor-list-header h1 {
```

Comment:
consider adding a class to the element instead of referencing by tag type. not a problem bc it works exactly the same here, just a best practice. i have been aiming to follow the [BEM standard ](https://getbem.com/naming/) in this project, but i am new to it.